[2015-11-27 14:25:03] default.ERROR: test exception in C:\xampp\htdocs\blivy\app\Queue\Job\TestJob.php:25
Backtrace: 
#0: C:\xampp\htdocs\blivy\app\Exception\Exception.php(26)Blivy\Exception\Exception->generateBacktrace()
#1: C:\xampp\htdocs\blivy\app\Exception\TemplateException.php(13)Blivy\Exception\Exception->__construct()
#2: C:\xampp\htdocs\blivy\app\Queue\Job\TestJob.php(25)Blivy\Exception\TemplateException->__construct()
#3: C:\xampp\htdocs\blivy\app\Queue\QueueWorker.php(32)Blivy\Queue\Job\TestJob->handle()
#4: C:\xampp\htdocs\blivy\blivy(47)Blivy\Queue\QueueWorker->tryJob()
#5: C:\xampp\htdocs\blivy\blivy(32)->queue_work()
  
[2015-11-27 14:25:08] default.ERROR: test exception in C:\xampp\htdocs\blivy\app\Queue\Job\TestJob.php:25
Backtrace: 
#0: C:\xampp\htdocs\blivy\app\Exception\Exception.php(26)Blivy\Exception\Exception->generateBacktrace()
#1: C:\xampp\htdocs\blivy\app\Exception\TemplateException.php(13)Blivy\Exception\Exception->__construct()
#2: C:\xampp\htdocs\blivy\app\Queue\Job\TestJob.php(25)Blivy\Exception\TemplateException->__construct()
#3: C:\xampp\htdocs\blivy\app\Queue\QueueWorker.php(32)Blivy\Queue\Job\TestJob->handle()
#4: C:\xampp\htdocs\blivy\blivy(47)Blivy\Queue\QueueWorker->tryJob()
#5: C:\xampp\htdocs\blivy\blivy(32)->queue_work()
  
[2015-11-27 14:25:11] default.ERROR: test exception in C:\xampp\htdocs\blivy\app\Queue\Job\TestJob.php:25
Backtrace: 
#0: C:\xampp\htdocs\blivy\app\Exception\Exception.php(26)Blivy\Exception\Exception->generateBacktrace()
#1: C:\xampp\htdocs\blivy\app\Exception\TemplateException.php(13)Blivy\Exception\Exception->__construct()
#2: C:\xampp\htdocs\blivy\app\Queue\Job\TestJob.php(25)Blivy\Exception\TemplateException->__construct()
#3: C:\xampp\htdocs\blivy\app\Queue\QueueWorker.php(32)Blivy\Queue\Job\TestJob->handle()
#4: C:\xampp\htdocs\blivy\blivy(47)Blivy\Queue\QueueWorker->tryJob()
#5: C:\xampp\htdocs\blivy\blivy(52)->queue_work()
#6: C:\xampp\htdocs\blivy\blivy(32)->queue_work()
  
[2015-11-27 14:25:23] default.ERROR: test exception in C:\xampp\htdocs\blivy\app\Queue\Job\TestJob.php:25
Backtrace: 
#0: C:\xampp\htdocs\blivy\app\Exception\Exception.php(26)Blivy\Exception\Exception->generateBacktrace()
#1: C:\xampp\htdocs\blivy\app\Exception\TemplateException.php(13)Blivy\Exception\Exception->__construct()
#2: C:\xampp\htdocs\blivy\app\Queue\Job\TestJob.php(25)Blivy\Exception\TemplateException->__construct()
#3: C:\xampp\htdocs\blivy\app\Queue\QueueWorker.php(32)Blivy\Queue\Job\TestJob->handle()
#4: C:\xampp\htdocs\blivy\blivy(47)Blivy\Queue\QueueWorker->tryJob()
#5: C:\xampp\htdocs\blivy\blivy(32)->queue_work()
  
[2015-11-27 14:25:28] default.ERROR: test exception in C:\xampp\htdocs\blivy\app\Queue\Job\TestJob.php:25
Backtrace: 
#0: C:\xampp\htdocs\blivy\app\Exception\Exception.php(26)Blivy\Exception\Exception->generateBacktrace()
#1: C:\xampp\htdocs\blivy\app\Exception\TemplateException.php(13)Blivy\Exception\Exception->__construct()
#2: C:\xampp\htdocs\blivy\app\Queue\Job\TestJob.php(25)Blivy\Exception\TemplateException->__construct()
#3: C:\xampp\htdocs\blivy\app\Queue\QueueWorker.php(32)Blivy\Queue\Job\TestJob->handle()
#4: C:\xampp\htdocs\blivy\blivy(47)Blivy\Queue\QueueWorker->tryJob()
#5: C:\xampp\htdocs\blivy\blivy(32)->queue_work()
  
[2015-11-27 14:25:31] default.ERROR: test exception in C:\xampp\htdocs\blivy\app\Queue\Job\TestJob.php:25
Backtrace: 
#0: C:\xampp\htdocs\blivy\app\Exception\Exception.php(26)Blivy\Exception\Exception->generateBacktrace()
#1: C:\xampp\htdocs\blivy\app\Exception\TemplateException.php(13)Blivy\Exception\Exception->__construct()
#2: C:\xampp\htdocs\blivy\app\Queue\Job\TestJob.php(25)Blivy\Exception\TemplateException->__construct()
#3: C:\xampp\htdocs\blivy\app\Queue\QueueWorker.php(32)Blivy\Queue\Job\TestJob->handle()
#4: C:\xampp\htdocs\blivy\blivy(47)Blivy\Queue\QueueWorker->tryJob()
#5: C:\xampp\htdocs\blivy\blivy(52)->queue_work()
#6: C:\xampp\htdocs\blivy\blivy(32)->queue_work()
  
[2015-11-27 14:25:49] default.ERROR: test exception in C:\xampp\htdocs\blivy\app\Queue\Job\TestJob.php:25
Backtrace: 
#0: C:\xampp\htdocs\blivy\app\Exception\Exception.php(26)Blivy\Exception\Exception->generateBacktrace()
#1: C:\xampp\htdocs\blivy\app\Exception\TemplateException.php(13)Blivy\Exception\Exception->__construct()
#2: C:\xampp\htdocs\blivy\app\Queue\Job\TestJob.php(25)Blivy\Exception\TemplateException->__construct()
#3: C:\xampp\htdocs\blivy\app\Queue\QueueWorker.php(32)Blivy\Queue\Job\TestJob->handle()
#4: C:\xampp\htdocs\blivy\blivy(47)Blivy\Queue\QueueWorker->tryJob()
#5: C:\xampp\htdocs\blivy\blivy(32)->queue_work()
  
[2015-11-27 14:25:52] default.ERROR: test exception in C:\xampp\htdocs\blivy\app\Queue\Job\TestJob.php:25
Backtrace: 
#0: C:\xampp\htdocs\blivy\app\Exception\Exception.php(26)Blivy\Exception\Exception->generateBacktrace()
#1: C:\xampp\htdocs\blivy\app\Exception\TemplateException.php(13)Blivy\Exception\Exception->__construct()
#2: C:\xampp\htdocs\blivy\app\Queue\Job\TestJob.php(25)Blivy\Exception\TemplateException->__construct()
#3: C:\xampp\htdocs\blivy\app\Queue\QueueWorker.php(32)Blivy\Queue\Job\TestJob->handle()
#4: C:\xampp\htdocs\blivy\blivy(47)Blivy\Queue\QueueWorker->tryJob()
#5: C:\xampp\htdocs\blivy\blivy(52)->queue_work()
#6: C:\xampp\htdocs\blivy\blivy(32)->queue_work()
  
[2015-11-27 14:26:05] default.ERROR: test exception in C:\xampp\htdocs\blivy\app\Queue\Job\TestJob.php:25
Backtrace: 
#0: C:\xampp\htdocs\blivy\app\Exception\Exception.php(26)Blivy\Exception\Exception->generateBacktrace()
#1: C:\xampp\htdocs\blivy\app\Exception\TemplateException.php(13)Blivy\Exception\Exception->__construct()
#2: C:\xampp\htdocs\blivy\app\Queue\Job\TestJob.php(25)Blivy\Exception\TemplateException->__construct()
#3: C:\xampp\htdocs\blivy\app\Queue\QueueWorker.php(32)Blivy\Queue\Job\TestJob->handle()
#4: C:\xampp\htdocs\blivy\blivy(47)Blivy\Queue\QueueWorker->tryJob()
#5: C:\xampp\htdocs\blivy\blivy(32)->queue_work()
  
[2015-11-27 14:26:08] default.ERROR: test exception in C:\xampp\htdocs\blivy\app\Queue\Job\TestJob.php:25
Backtrace: 
#0: C:\xampp\htdocs\blivy\app\Exception\Exception.php(26)Blivy\Exception\Exception->generateBacktrace()
#1: C:\xampp\htdocs\blivy\app\Exception\TemplateException.php(13)Blivy\Exception\Exception->__construct()
#2: C:\xampp\htdocs\blivy\app\Queue\Job\TestJob.php(25)Blivy\Exception\TemplateException->__construct()
#3: C:\xampp\htdocs\blivy\app\Queue\QueueWorker.php(32)Blivy\Queue\Job\TestJob->handle()
#4: C:\xampp\htdocs\blivy\blivy(47)Blivy\Queue\QueueWorker->tryJob()
#5: C:\xampp\htdocs\blivy\blivy(52)->queue_work()
#6: C:\xampp\htdocs\blivy\blivy(32)->queue_work()
  
[2015-11-27 14:26:22] default.ERROR: test exception in C:\xampp\htdocs\blivy\app\Queue\Job\TestJob.php:25
Backtrace: 
#0: C:\xampp\htdocs\blivy\app\Exception\Exception.php(26)Blivy\Exception\Exception->generateBacktrace()
#1: C:\xampp\htdocs\blivy\app\Exception\TemplateException.php(13)Blivy\Exception\Exception->__construct()
#2: C:\xampp\htdocs\blivy\app\Queue\Job\TestJob.php(25)Blivy\Exception\TemplateException->__construct()
#3: C:\xampp\htdocs\blivy\app\Queue\QueueWorker.php(32)Blivy\Queue\Job\TestJob->handle()
#4: C:\xampp\htdocs\blivy\blivy(49)Blivy\Queue\QueueWorker->tryJob()
#5: C:\xampp\htdocs\blivy\blivy(34)->queue_work()
  
[2015-11-27 14:26:25] default.ERROR: test exception in C:\xampp\htdocs\blivy\app\Queue\Job\TestJob.php:25
Backtrace: 
#0: C:\xampp\htdocs\blivy\app\Exception\Exception.php(26)Blivy\Exception\Exception->generateBacktrace()
#1: C:\xampp\htdocs\blivy\app\Exception\TemplateException.php(13)Blivy\Exception\Exception->__construct()
#2: C:\xampp\htdocs\blivy\app\Queue\Job\TestJob.php(25)Blivy\Exception\TemplateException->__construct()
#3: C:\xampp\htdocs\blivy\app\Queue\QueueWorker.php(32)Blivy\Queue\Job\TestJob->handle()
#4: C:\xampp\htdocs\blivy\blivy(49)Blivy\Queue\QueueWorker->tryJob()
#5: C:\xampp\htdocs\blivy\blivy(54)->queue_work()
#6: C:\xampp\htdocs\blivy\blivy(34)->queue_work()
  
[2015-11-27 14:26:50] default.ERROR: test exception in C:\xampp\htdocs\blivy\app\Queue\Job\TestJob.php:25
Backtrace: 
#0: C:\xampp\htdocs\blivy\app\Exception\Exception.php(26)Blivy\Exception\Exception->generateBacktrace()
#1: C:\xampp\htdocs\blivy\app\Exception\TemplateException.php(13)Blivy\Exception\Exception->__construct()
#2: C:\xampp\htdocs\blivy\app\Queue\Job\TestJob.php(25)Blivy\Exception\TemplateException->__construct()
#3: C:\xampp\htdocs\blivy\app\Queue\QueueWorker.php(32)Blivy\Queue\Job\TestJob->handle()
#4: C:\xampp\htdocs\blivy\blivy(50)Blivy\Queue\QueueWorker->tryJob()
#5: C:\xampp\htdocs\blivy\blivy(35)->queue_work()
  
[2015-11-27 14:27:06] default.ERROR: test exception in C:\xampp\htdocs\blivy\app\Queue\Job\TestJob.php:25
Backtrace: 
#0: C:\xampp\htdocs\blivy\app\Exception\Exception.php(26)Blivy\Exception\Exception->generateBacktrace()
#1: C:\xampp\htdocs\blivy\app\Exception\TemplateException.php(13)Blivy\Exception\Exception->__construct()
#2: C:\xampp\htdocs\blivy\app\Queue\Job\TestJob.php(25)Blivy\Exception\TemplateException->__construct()
#3: C:\xampp\htdocs\blivy\app\Queue\QueueWorker.php(32)Blivy\Queue\Job\TestJob->handle()
#4: C:\xampp\htdocs\blivy\blivy(48)Blivy\Queue\QueueWorker->tryJob()
#5: C:\xampp\htdocs\blivy\blivy(33)->queue_work()
  
[2015-11-27 14:27:23] default.ERROR: test exception in C:\xampp\htdocs\blivy\app\Queue\Job\TestJob.php:25
Backtrace: 
#0: C:\xampp\htdocs\blivy\app\Exception\Exception.php(26)Blivy\Exception\Exception->generateBacktrace()
#1: C:\xampp\htdocs\blivy\app\Exception\TemplateException.php(13)Blivy\Exception\Exception->__construct()
#2: C:\xampp\htdocs\blivy\app\Queue\Job\TestJob.php(25)Blivy\Exception\TemplateException->__construct()
#3: C:\xampp\htdocs\blivy\app\Queue\QueueWorker.php(32)Blivy\Queue\Job\TestJob->handle()
#4: C:\xampp\htdocs\blivy\blivy(50)Blivy\Queue\QueueWorker->tryJob()
#5: C:\xampp\htdocs\blivy\blivy(33)->queue_work()
  
[2015-11-27 14:27:26] default.ERROR: test exception in C:\xampp\htdocs\blivy\app\Queue\Job\TestJob.php:25
Backtrace: 
#0: C:\xampp\htdocs\blivy\app\Exception\Exception.php(26)Blivy\Exception\Exception->generateBacktrace()
#1: C:\xampp\htdocs\blivy\app\Exception\TemplateException.php(13)Blivy\Exception\Exception->__construct()
#2: C:\xampp\htdocs\blivy\app\Queue\Job\TestJob.php(25)Blivy\Exception\TemplateException->__construct()
#3: C:\xampp\htdocs\blivy\app\Queue\QueueWorker.php(32)Blivy\Queue\Job\TestJob->handle()
#4: C:\xampp\htdocs\blivy\blivy(50)Blivy\Queue\QueueWorker->tryJob()
#5: C:\xampp\htdocs\blivy\blivy(33)->queue_work()
  
[2015-11-27 14:27:41] default.ERROR: test exception in C:\xampp\htdocs\blivy\app\Queue\Job\TestJob.php:25
Backtrace: 
#0: C:\xampp\htdocs\blivy\app\Exception\Exception.php(26)Blivy\Exception\Exception->generateBacktrace()
#1: C:\xampp\htdocs\blivy\app\Exception\TemplateException.php(13)Blivy\Exception\Exception->__construct()
#2: C:\xampp\htdocs\blivy\app\Queue\Job\TestJob.php(25)Blivy\Exception\TemplateException->__construct()
#3: C:\xampp\htdocs\blivy\app\Queue\QueueWorker.php(32)Blivy\Queue\Job\TestJob->handle()
#4: C:\xampp\htdocs\blivy\blivy(50)Blivy\Queue\QueueWorker->tryJob()
#5: C:\xampp\htdocs\blivy\blivy(35)->queue_work()
  
[2015-11-27 14:27:55] default.ERROR: test exception in C:\xampp\htdocs\blivy\app\Queue\Job\TestJob.php:25
Backtrace: 
#0: C:\xampp\htdocs\blivy\app\Exception\Exception.php(26)Blivy\Exception\Exception->generateBacktrace()
#1: C:\xampp\htdocs\blivy\app\Exception\TemplateException.php(13)Blivy\Exception\Exception->__construct()
#2: C:\xampp\htdocs\blivy\app\Queue\Job\TestJob.php(25)Blivy\Exception\TemplateException->__construct()
#3: C:\xampp\htdocs\blivy\app\Queue\QueueWorker.php(32)Blivy\Queue\Job\TestJob->handle()
#4: C:\xampp\htdocs\blivy\blivy(51)Blivy\Queue\QueueWorker->tryJob()
#5: C:\xampp\htdocs\blivy\blivy(36)->queue_work()
  
[2015-11-27 14:28:00] default.ERROR: test exception in C:\xampp\htdocs\blivy\app\Queue\Job\TestJob.php:25
Backtrace: 
#0: C:\xampp\htdocs\blivy\app\Exception\Exception.php(26)Blivy\Exception\Exception->generateBacktrace()
#1: C:\xampp\htdocs\blivy\app\Exception\TemplateException.php(13)Blivy\Exception\Exception->__construct()
#2: C:\xampp\htdocs\blivy\app\Queue\Job\TestJob.php(25)Blivy\Exception\TemplateException->__construct()
#3: C:\xampp\htdocs\blivy\app\Queue\QueueWorker.php(32)Blivy\Queue\Job\TestJob->handle()
#4: C:\xampp\htdocs\blivy\blivy(51)Blivy\Queue\QueueWorker->tryJob()
#5: C:\xampp\htdocs\blivy\blivy(36)->queue_work()
  
[2015-11-27 14:28:03] default.ERROR: test exception in C:\xampp\htdocs\blivy\app\Queue\Job\TestJob.php:25
Backtrace: 
#0: C:\xampp\htdocs\blivy\app\Exception\Exception.php(26)Blivy\Exception\Exception->generateBacktrace()
#1: C:\xampp\htdocs\blivy\app\Exception\TemplateException.php(13)Blivy\Exception\Exception->__construct()
#2: C:\xampp\htdocs\blivy\app\Queue\Job\TestJob.php(25)Blivy\Exception\TemplateException->__construct()
#3: C:\xampp\htdocs\blivy\app\Queue\QueueWorker.php(32)Blivy\Queue\Job\TestJob->handle()
#4: C:\xampp\htdocs\blivy\blivy(51)Blivy\Queue\QueueWorker->tryJob()
#5: C:\xampp\htdocs\blivy\blivy(56)->queue_work()
#6: C:\xampp\htdocs\blivy\blivy(36)->queue_work()
  
